spec/mock-request.js

```diff
@@ -227,6 +251,16 @@ ExpectedRequest.prototype = {
             err: err,
         };
     },
+
+    /**
+     * Timeout a request by waitMs.
+     *
+     * @param {Number} waitMs number of miliseconds of timeout.
+     */
+    timeout: function(waitMs) {
```

Comment:
What this actually does is advance the amount of time this HTTP request has taken - this may or may not result in a timeout. Can you please clarify this in the docs, else it appears as if you're saying "time out the request after `waitMs`".